### PR TITLE
Remove useless `permission`

### PR DESCRIPTION
**Pull request by glasserc**

### CODE REVIEW

**Comment by leplatrem** [comment]
So this will work for anynomous users too?

**Comment by glasserc** [comment]
It will work the same as it does today (i.e. no).

**Comment by glasserc** [comment]
Sorry, let me rephrase... this line contributes nothing to whether this endpoint works for anonymous users or not. That's controlled elsewhere. At present, that setting is for the endpoint to allow anonymous access.
